File: 3st/Jaeyoung/마지막 두 원소.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> num_list) {
    vector<int> answer = num_list;
    int size = num_list.size();
    answer.push_back(num_list[size-1] > num_list[size-2]
                     ? num_list[size-1] - num_list[size-2]
                     : num_list[size-1]*2);
    return answer;
}

File: 4st/Jisoo/programmer/마지막 두 원소.java
import java.util.Arrays;
class Solution {
    public int[] solution(int[] num_list) {
        int a = num_list[num_list.length - 2], b = num_list[num_list.length - 1];
        int[] answer = Arrays.copyOf(num_list, num_list.length + 1);
        answer[answer.length - 1] = b > a ? b - a : b * 2;
        return answer;
    }
}